src/grelu/model/blocks.py

```diff
@@ -697,22 +698,38 @@ def __init__(
         pos_dropout: float,
         attn_dropout: float,
         ff_dropout: float,
+        flash_attn: bool,
         dtype=None,
         device=None,
     ) -> None:
         super().__init__()
         self.norm = Norm("layer", in_len)
-        self.mha = Attention(
-            in_len=in_len,
-            n_heads=n_heads,
-            n_pos_features=n_pos_features,
-            key_len=key_len,
-            value_len=value_len,
-            pos_dropout=pos_dropout,
-            attn_dropout=attn_dropout,
-            dtype=dtype,
-            device=device,
-        )
+
+        if flash_attn:
+
+            print(
+                "WARNING: FlashAttention does not use pos_dropout, key_len, value_len, n_pos_features arguments. \
```

Comment:
Let's use warnings.warn() instead of printing "WARNING"

Reply:
done